pkg/internalregistry/controllers/service_account_controller.go

```diff
@@ -195,61 +229,7 @@ func (c *serviceAccountController) sync(ctx context.Context, key string) error {
 		}
 	}
 
-	// nothing else to do if we are not dealing with a bound image pull secret
-	if secret.Annotations[InternalRegistryAuthTokenTypeAnnotation] != AuthTokenTypeBound {
-		return nil
-	}
```

Comment:
I don't see this logic captured above - was this removal intentional?

Reply:
Yes, this was intentional.